src/content/protocols/kodiak-v3.md

```diff
@@ -181,14 +184,19 @@ No external dependency has been found.
 
-# Security Council
-
-| Multisig / Role | Address                                    | Type         | At least 7 signers | At least 51% threshold | ≥50% non-insider signers | Signers publicly announced |
-| --------------- | ------------------------------------------ | ------------ | ------------------ | ---------------------- | ------------------------ | -------------------------- |
-| No 1            | 0x21802b7C3DF57e98df45f1547b0F1a72F2CD1aED | Multisig 5/9 |                    |                        |                          |                            |
-| No 2            | 0x7D8A57bf453ecf440093fe6484c78B91b377e638 | Multisig 3/6 |                    |                        |                          |                            |
-| No 3            | 0x8A98f7A3dF97029bd8992966044a06DEe4CC5299 | Multisig 2/3 |                    |                        |                          |                            |
-| No 4            | 0xD13948F99525FB271809F45c268D72a3C00a568D | Multisig 3/5 |                    |                        |                          |                            |
+## Risk Parameter Control
 
+| Multisig / Role        | Address                                    | Type         | At least 7 signers | At least 51% threshold | ≥50% non-insider signers | Signers publicly announced |
+| ---------------------- | ------------------------------------------ | ------------ | ------------------ | ---------------------- | ------------------------ | -------------------------- |
+| Farming parameters     | 0x21802b7C3DF57e98df45f1547b0F1a72F2CD1aED | Multisig 2/4 | ❌                  | ❌                      | ❌                        | ❌                          |
+| Liquidity parameters   | 0x7D8A57bf453ecf440093fe6484c78B91b377e638 | Multisig 3/6 | ❌                  | ❌                      | ❌                        | ❌                          |
+| PandasToken parameters | 0x8A98f7A3dF97029bd8992966044a06DEe4CC5299 | Multisig 2/3 | ❌                  | ✅                      | ❌                        | ❌                          |
+| RewardVault parameters | 0xD13948F99525FB271809F45c268D72a3C00a568D | Multisig 3/5 | ❌                  | ✅                      | ❌                        | ❌                          |
 
 
+| Role               | Risk Parameter | Controlled by | Guardrails enforced by | Contract            |
+| ------------------ | -------------- | ------------- | ---------------------- | ------------------- |
+| Farming parameters |                | Multisig 2/4  |                        | UniswapV3Factory    |
+|                    |                | Multisig 3/6  |                        | KodiakIslandFactory |
+|                    |                | Multisig 2/3  |                        | PandaFactory        |
+|                    |                | Multisig 3/5  |                        | RewardVaultFactory  |
```

Comment:
@newgnart sorry this is maybe my mistake, but this section is not necessary, the comparable section in the aave v3 report, is unique to aave bc of their permission design, I dont think it's needed here